examples/examples_py_test.go

```diff
@@ -39,6 +39,22 @@ func TestAccAwsProfilePy(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestAccAwsProfileRolePy(t *testing.T) {
+	test := getPythonBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir: path.Join(getCwd(t), "aws-profile-role-py"),
+			// TODO: Temporarily skip the extra runtime validation due to test failure.
```

Comment:
Is there a GH issue tracking this?

Reply:
lol, oh, looks like this has been a known issue seeing it commented out in a bunch of other tests... :(